pftp/handle_commands.go

```diff
@@ -161,8 +163,8 @@ func (c *clientHandler) handlePBSZ() *result {
 
 // response PROT to client and store command line when connect by TLS & not loggined
 func (c *clientHandler) handlePROT() *result {
-	if c.controlInTLS {
-		if !c.isLoggedin {
+	if atomic.LoadInt32(&c.controlInTLS) == 1 {
```

Comment:
you had better to make control structs, like a https://gist.github.com/mahan/6256149

Reply:
I looks easier for code read!

Reply:
https://github.com/pyama86/pftp/pull/45/commits/0782994610d7ae275b46b10769361c23a6eabb30

replace atomic operation to abool module.